installing/installing_with_agent/installing-with-agent.adoc

```diff
@@ -0,0 +1,24 @@
+:_content-type: ASSEMBLY
+[id="installing-with-agent"]
+= Installing with Agent
+include::_attributes/common-attributes.adoc[]
+:context: installing-with-agent
+
+toc::[]
+
+== Agent installation
+Agent installation is based on the technology of Assisted Installer (AI), that is link:https://console.redhat.com[console.redhat.com] and the Infrastructure Operator. It is a command line tool that generates an image that runs the Assisted Service and the Assisted Installer Agent.
```

Comment:
@dhellmann so can i re-phrase that though the technology is based on AI, access is not required as a pre-requisite.

Reply:
I'm also worried that saying the tool generates an image that runs those services gives the impression that the image can be used for multiple clusters and that the user should expect to interact with the services, but neither of those things is true. We use the assisted installer tech, but add automation so that the user only has to boot the image to get their cluster. That automation the most important aspect of the new tool.